db/migrate/20230921021458_add_user_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
same here likes already has the user_id column, this result in `PG::DuplicateColumn: ERROR: `